lib/versioned/^3.7.0/taskTree.js

```diff
@@ -4,14 +4,22 @@ module.exports = function(tasks) {
   var map = {};
   var arr = [];
   Object.keys(tasks).forEach(function(taskname) {
-    var task = { label: taskname, nodes: [], };
+    var task = {
+      label: taskname,
+      type: 'task',
```

Comment:
@sttk why did you add `type` to this structure?

Reply:
For getting same formatted json data by `--tasks-json` with gulp v4.

An example of json data is as follows:
```json
{"label":"Tasks for ~/project/gulpfile.js","nodes":[
    {"label":"taskA","type":"task","nodes":[
        {"label":"<parallel>","type":"function","branch":true,"nodes": [ ...
```